src/main/java/org/popper/gherkin/GherkinExtension.java

```diff
@@ -53,18 +53,19 @@ public void beforeAll(ExtensionContext context) throws Exception {
 
     @Override
     public void beforeEach(ExtensionContext context) throws Exception {
-        getOrCreateRunner(context).startMethod(context.getRequiredTestInstance(), context.getRequiredTestMethod());
+        getOrCreateRunner(context).startMethod(context, context.getRequiredTestInstance(),
+                context.getRequiredTestMethod());
     }
 
     @Override
     public void afterEach(ExtensionContext context) throws Exception {
-        getOrCreateRunner(context).endMethod(context.getRequiredTestInstance(), context.getRequiredTestMethod(),
-                context.getExecutionException());
+        getOrCreateRunner(context).endMethod(context, context.getRequiredTestInstance(),
+                context.getRequiredTestMethod(), context.getExecutionException());
```

Comment:
Just providing the parameter context would be fine here

Reply:
Ok. Fixed. Hope, if've done right with github.